src/components/Layout/Header/NavBar/MobileNavBar.tsx

```diff
@@ -77,22 +165,106 @@ export const MobileNavBar = memo(() => {
     }
   }, [])
 
+  const handleQrCodeClick = useCallback(() => {
+    onClose()
```

Comment:
Any reason why we have this as a first call in all event handlers?

Reply:
Its to close the dialog

Reply:
Or actually that makes sense - just thinking `<ActionMenuButton />` should perhaps handle this so we don't have to do it here?

Reply:
Not sure I follow how that would work. I like exposing exactly whats happening on the onClick instead of hiding it away in another component. 

Its always possible we may tap a button and not want it to close, but at the moment don't see that happening.

Reply:
Discussed this one out, agreed to do it as a ☕ follow-up